keras_cv/models/segmentation/fcn_test.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+from keras_cv import models
+from keras_cv.models import segmentation
+
+
+class FCNTest(tf.test.TestCase):
+    def test_fcn_model_with_vgg16_backbone_construction_with_preconfigured_setting(
+        self,
+    ):
+        model = segmentation.FCN(
+            classes=11, backbone="vgg16", model_architecture="fcn8s"
+        )
+        input_image = tf.random.uniform(shape=[2, 256, 256, 3])
```

Comment:
Possibly redundant suggestion - try using smaller shapes here, such as `[1, 64, 64, 3]`. This makes the test run computationally cheaper and proves the same point :)

Since there are ~1000 tests in KCV, we could save some percentage of time/compute on all of them by preferring smaller inputs. Thoughts? @tanzhenyu @bhack

Reply:
Perhaps, yes.

Reply:
yeah, actually we reduced image shape / backbone size in some of our recent unit tests, because it gives OOM in Window test

Reply:
Should we go and reduce them in previous tests too? If we can get the test cases to run in 30% less time, that's significant as they run on each commit in each PR.

Reply:
Could you update this?

Reply:
Yeah !